artsy/force
# feat: unauthenticated users can start Identity Verification

The type of this PR is: **Feature**





This PR solves [PLATFORM-4165](https://artsyproduct.atlassian.net/browse/PLATFORM-4165)

### Description



#### Start identity verification process without an Artsy account.

This PR decouples identity verification from the _User_ and allows access to the page without a login. It depends on the following [MP PR](https://github.com/artsy/metaphysics/pull/4133) for the flow to work and _should be merged after MP changes are deployed_.

* decouple IdentityVerificationApp from `Me`
* open access to /identity-verification route(s)
* update tests and remove no longer used component
(more in comments 👇)

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
